.travis.yml

```diff
@@ -35,17 +33,10 @@ before_script:
 
 script:
     - python continuous_integration/show-python-packages-versions.py
-    # We want to back out of the current working directory to make
-    # sure we are using nilearn installed in site-packages rather
-    # than the one from the current working directory
-    # Parentheses (run in a subshell) are used to leave
-    # the current directory unchanged
     - make test
 
 after_success:
     # Ignore coveralls failures as the coveralls server is not very reliable
     # but we don't want travis to report a failure in the github UI just
     # because the coverage report failed to be published.
-    # coveralls need to be run from the git checkout
-    # so we need to copy the coverage results from TEST_RUN_FOLDER
-    - if [[ "$COVERAGE" == "true" ]]; then cp "$TEST_RUN_FOLDER/.coverage" .; coveralls || echo "failed"; fi
+    - if [[ "$COVERAGE" == "true" ]]; then coveralls || echo "failed"; fi
```

Comment:
Is COVERAGE ever true? I am not sure that we are using coverage with the scipy-lectures.